<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelo340</class>
 <widget class="QDialog" name="modelo340">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MODELO 340</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>NIF:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="niflineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Núm. presentación:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numerospinBox">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>EJERCICIO:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ejerciciocomboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Apellidos y nombre o razón social:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nombrelineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Período:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="periodocomboBox">
         <item>
          <property name="text">
           <string>1T Primer trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2T Segundo trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3T Tercer trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4T Cuarto trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01 Enero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02 Febrero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03 Marzo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04 Abril</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05 Mayo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06 Junio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07 Julio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08 Agosto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09 Septiembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 Octubre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11 Noviembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 Diciembre</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="fechagroupBox">
       <property name="title">
        <string>Selección de fechas</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>INICIAL:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="inicialdateEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>FINAL:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="finaldateEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="complgroupBox">
       <property name="title">
        <string>Complementaria / Sustitutiva</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="sustitutivaradioButton">
          <property name="text">
           <string>Sustitutiva</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="complementariaradioButton">
          <property name="text">
           <string>Complementaria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Núm.declaración anterior:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="anteriorlineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Persona con quien relacionarse</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Teléfono:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="telefonolineEdit">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Apellidos y nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nombrerellineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Generar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>modelo340</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
